### PR TITLE
Use a separate email address for the Volunteer page.

### PR DESCRIPTION
**Pull request by seanlip**

### CODE REVIEW

File: app/core/domain/email_manager_test.py [hunk 1]
```diff
@@ -24,14 +24,33 @@ class SendEmailToAdminTest(app_engine_test_base.GenericTestBase):
     """Test for email to be sent to admin email address."""
 
     def test_send_mail_to_admin(self):
-        """Test send_mail_to_admin sends mail to admin email address."""
+        """Test send_mail sends mail to admin email address."""
```
Comment (hoangviet1993) [inline app/core/domain/email_manager_test.py]: This test only works with the send_mail_to_admin method right?

File: app/core/controllers/outgoing_emails.py [hunk 2]
```diff
@@ -89,7 +89,12 @@ def post(self):
         email_contents = ForwardToAdminEmailHandler.write_email_contents(
```
Comment (hoangviet1993) [inline app/core/controllers/outgoing_emails.py]: Nit: Should we change this to `ForwardToEmailHandler`?

File: app/core/controllers/outgoing_emails_test.py [hunk 3]
```diff
@@ -106,31 +106,39 @@ def test_email_is_forwarded(self):
...
+        self.assertEqual(1, len(messages))
 
         form_contents = {
             'email_type': config.EMAIL_TYPE_PARTNERSHIPS,
             'organization': 'Non-profit organization 1',
             'email': 'user1@example.com',
             'comment': 'We are looking for partners.'
         }
-
-        self.assertEqual(2, len(messages))
```
Comment (hoangviet1993) [inline app/core/controllers/outgoing_emails_test.py]: Thanks for this :)

File: app/core/domain/email_manager_test.py (hunk 1 above)
Comment (seanlip) [reply app/core/domain/email_manager_test.py]: Ah yes. I've fixed the comment, thanks!

File: app/core/controllers/outgoing_emails.py (hunk 2 above)
Comment (seanlip) [reply app/core/controllers/outgoing_emails.py]: I thought about it, but I didn't want to give the impression that this forwards to arbitrary email addresses. Instead, I changed the docstring of the handler to be clearer about what sorts of email addresses the handler actually forwards to.

File: app/core/controllers/outgoing_emails_test.py (hunk 3 above)
Comment (seanlip) [reply app/core/controllers/outgoing_emails_test.py]: You're welcome :P

**Pull request merged**